### PR TITLE
Az452 integrate search with riak

### PR DESCRIPTION
**Pull request by rzezeski**
Moving forward Search will come bundled w/ Riak as a feature that can be enabled (not by default).

See also https://github.com/basho/riak_search/pull/63

### CODE REVIEW

File: rel/vars/dev2_vars.config [hunk 1]
```diff
@@ -15,6 +15,11 @@
 {sasl_log_dir,      "log/sasl"}.
 {mapred_queue_dir, "data/mr_queue"}.
 
+%% riak_search
+{merge_index_data_root,  "data/merge_index"}.
+{analyzer_conns,    50}.
+{analyzer_port,     6095}.
```
Comment (Vagabond) [inline rel/vars/dev2_vars.config:21]: Shouldn't the analyzer port be different across the 3 different dev VM configs so they don't conflict?

File: tests/riak_search/_files/books/books.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<add>
```
Comment (Vagabond) [inline tests/riak_search/_files/books/books.xml:1]: Why aren't all these test files bundled with riak_search?

File: rel/vars/dev2_vars.config (hunk 1 above)
Comment (rzezeski) [inline rel/vars/dev2_vars.config:21]: They should, but considering this goes away w/ qilr I'll just remove it in the Qilr patch

File: tests/riak_search/_files/books/books.xml (hunk 2 above)
Comment (rzezeski) [inline tests/riak_search/_files/books/books.xml:1]: Because the only way to run them is with an instance of Search running, which is now part of Riak.

**Comment by Vagabond** [comment]
+1 to merge

**Pull request merged**